src/main/java/edu/cornell/kfs/module/purap/batch/dataaccess/impl/JaggaerUploadDaoJdbc.java

```diff
@@ -245,7 +245,7 @@ private CuSqlChunk buildPurchaseOrderLimitSubQuery(Date processingDate) {
                 "FROM KFS.PUR_PO_T ",
                 "WHERE PO_LST_TRNS_DT > ",
                 CuSqlChunk.forParameter(Types.DATE, processingDate),
-                " GROUP BY VNDR_HDR_GNRTD_ID HAVING COUNT(1) > 1)");
+                " GROUP BY VNDR_HDR_GNRTD_ID HAVING COUNT(1) >= 1)");
```

Comment:
changing this condition from `> 1` to `>= 1` might make the whole `HAVING` clause redundant, though I agree that writing it this way seems to make the business logic clearer. Please let me know if you'd rather leave it like this or if you think it should be changed.

Reply:
I think the improved clarity of leaving the code like this is worth it, would you agree?

Reply:
Leaving it like this is fine with me. Thanks for confirming!